### PR TITLE
[BLOG] Add what-do-we-need-to-write.md

### PR DESCRIPTION
**Pull request by MarsBarLee**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:
no labs links found
no llc links found

### CODE REVIEW

**Comment by noatamir** [comment]
@gabalafou CC @trallard 
- this blog is failing on missing author. 
- All the authors are in storyblok and have images. 
- I am not sure if I should put multiple authors in brackets or quotes 
- There is no guidance in the readme.

File: apps/labs/posts/what-do-we-need-to-write.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: 'Why is writing blog posts hard?'
+published: April 10, 2022
+author: tania-allard, melissa-mendonca, isabela-presedo-floyd, rohit-goswami, tony-fast, dillon-roach, dharhas-pothina, eric-kelly
```
Comment (gabalafou) [inline apps/labs/posts/what-do-we-need-to-write.md:4]: @noatamir you're already aware of this, I think, but multiple authors seems not supported at the moment. This line causes the build to fail. I'm not sure what we should do. cc @trallard

**Review by gabalafou — COMMENTED** [review]
This PR is almost ready to merge. Just have to fix the issue with multiple authors.

File: apps/labs/posts/what-do-we-need-to-write.md (hunk 1 above)
Comment (gabalafou) [inline apps/labs/posts/what-do-we-need-to-write.md:4]:
```suggestion
author: isabela-presedo-floyd
```

**Review by gabalafou — CHANGES_REQUESTED** [review]
I suggest that we modify the author field to just have Isabela and push the other authors to one line at the beginning.

File: apps/labs/posts/what-do-we-need-to-write.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+category: [Community, Developer workflows]
+featuredImage:
+  src: /posts/what-do-we-need-to-write/blog_feature_var2.svg
+  alt: 'An illustration of a brown and a white hand coming towards each other to pass a business card with the logo of Quansight Labs'
+hero:
+  imageSrc: /posts/what-do-we-need-to-write/blog_hero_var2.svg
+  imageAlt: 'An illustration of a black hand holding up a microphone, with some graphical elements highlighting the top of the microphone'
+---
+
+Additional authors: Tania Allard, Melissa Mendonça, Rohit Goswami, Tony Fast, Dillon Roach, Dharhas Pothina, Eric Kelly.
```
Comment (gabalafou) [inline apps/labs/posts/what-do-we-need-to-write.md:17]:
```suggestion
Additional authors: Tania Allard, Melissa Mendonça, Rohit Goswami, Tony Fast, Dillon Roach, 
Dharhas Pothina, Eric Kelly.

```

**Pull request merged**